packages/block-library/src/template-part/theme.scss

```diff
@@ -1,9 +1,7 @@
 // Same as the group block styles.
-:where(.wp-block-template-part) {
-	&.has-background {
```

Comment:
It seems we previously did have some specificity left here. We should make sure it tests well now.